test/testdata/parser/ruby_25.rb

```diff
@@ -8,10 +8,9 @@ def toto
         end
     end
 end
-  
```

Comment:
why was this affected?

Reply:
Just whitespace change. (`sed` likes having newline at EOF). I isolated whitespace-only changes to a separate commit, so I can remove if you would like.